cmd/broadcaster-cli/app/utxos/create/create.go

```diff
@@ -79,7 +79,7 @@ var Cmd = &cobra.Command{
 
 		names := helper.GetOrderedKeys(keySetsMap)
 
-		wocClient := woc_client.New(!isTestnet, woc_client.WithAuth(wocApiKey), woc_client.WithLogger(logger))
+		wocClient := wocclient.New(!isTestnet, wocclient.WithAuth(wocAPIKey), wocclient.WithLogger(logger))
```

Comment:
I think `woc_client` could remain as it is, not?

Reply:
Done